src/applications/simple-forms/21-0966/containers/ConfirmationPage.jsx

```diff
@@ -121,7 +125,12 @@ export const ConfirmationPage = props => {
                 <p>
                   Our records show that you already have an intent to file for
                   pension claims. Your intent to file for pension claims expires
-                  on {data['view:activePensionITF'].expirationDate}.
+                  on{' '}
+                  {format(
```

Comment:
what's the difference between the dates that have `format` and the ones that don't?

Reply:
Theoretically they should all be formatted. Are you seeing one that isn't?

Reply:
The two individual benefit Yes/No pages have formatted expiration dates and the confirmation page (and its helpers) should have a few too.